changelogs/fragments/497-s3_sync-add-storage_class.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- s3_sync: new storage_class feature allowing to specify the storage_class when any object is added to an s3_bucket (https://github.com/ansible-collections/community.aws/issues/358).
```

Comment:
```suggestion
- s3_sync - new ``storage_class`` feature allowing to specify the storage class when any object is added to an S3 bucket (https://github.com/ansible-collections/community.aws/issues/358).
```

Reply:
@tremble Fixed. Thank you for reviewing.